src/styles/blocks/clock.scss

```
@@ -0,0 +1,67 @@
+.clock {
+  display: flex;
+  position: relative;
+  &__pin {
+    @include Square(20px);
+    position: absolute;
+    background-color: black;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    border-radius: 50%;
+    z-index: 2;
+  }
+
+  &__image {
+    @include Square($cloak-size);
+    position: relative;
+    transform: translate(-50%,-50%);
+    top: 50%;
+    left: 50%;
+  }
+
+  &__hand {
+    @include ClockHand;
+
+    &--1 {
+      height: 4px;
+      width: 25%;
+      border-top-right-radius: 50%;
+      border-bottom-right-radius: 50%;
+      background-color: red;
+      animation: ClockHandSeconds 60s linear infinite;
+      transform-origin: 2px 2px;
+    }
+
+    &--2 {
+      height: 15%;
+      width: 4px;
+      border-bottom-left-radius: 50%;
+      border-bottom-right-radius: 50%;
+      background-color: blue;
+
+      animation: ClockHandMinutes 3600s linear infinite;
+      transform-origin: 2px 2px;
+    }
+  }
+}
+
+@keyframes ClockHandSeconds {
+  0% {
+    transform: rotate(-90deg);
+  }
+
+  100% {
+    transform: rotate(270deg);
+  }
+}
+
+@keyframes ClockHandMinutes {
+  0% {
+    transform: rotate(-180deg);
+  }
+
+  100% {
+    transform: rotate(180deg);
+  }
```

2. And add :hover to pause animation
```suggestion
&:hover {
  &__hand {
  animation-play-state: paused;
}
```